docker/runtime/python-2.7/uwsgi.ini

```diff
@@ -0,0 +1,11 @@
+[uwsgi]
+plugins-dir = /usr/lib/uwsgi
+need-plugin = python
+pythonpath = /usr/local/lib/python2.7
+pythonpath = /usr/local/lib/python2.7/site-packages
```

Comment:
Oh good call, I haven't tested a function that uses requirements.  The above path is sufficient to pick up "pip install ..." modules from Dockerfile, but I will test explicitly against function-provided requirements tomorrow...